v-next/hardhat-keystore/src/internal/ui/request-secret-input.ts

```diff
@@ -0,0 +1,53 @@
+import { HardhatError } from "@ignored/hardhat-vnext-errors";
+
+export async function requestSecretInput(
```

Comment:
For a follow-up PR: I think we should move this to utils, we have similar things in a few places, including one implementation that uses enquirer.